docs/website/static/security.html

```diff
@@ -0,0 +1,157 @@
+          </div>
+          <h2 class="divider line glow">Reporting a Security Bug</h2>
+          <div class="banner-subcontent">
+            <ul>
+              <li>If you think you have found a security issue in an OPA project, please send an
+                email to <a href="mailto:open-policy-agent-security@googlegroups.com">open-policy-agent-security</a>.
+                This list is delivered to a small security team. We will then acknowledge receipt
+                of your report and prioritize initial analysis of severity.</li>
+
+              <li>After the initial reply to your report, the security team will endeavor to keep you informed
+                of the progress being made towards a fix and full announcement, and may ask for additional
+                information or guidance surrounding the reported issue.</p>
+
+              <li>If you have not received a reply to your report within two days, please reach out on our
+               <a href="https://slack.openpolicyagent.org/">Slack</a>
+               by posting a message in the <code>#contributors</code> channel.</li>
+
+              <li>Note that the <code>#conributors</code> channel is public, so please don't discuss details of your
+                issue there. Instead, simply say that you're trying to get a hold of someone from the security team.</li>
+            </ul>
+          </div>
+          <h2 class="divider line glow">Disclosure Policy</h2>
+          <div class="banner-subcontent">
+            <ol>
+              <li>The security report is received and is assigned a primary handler.
+                This person will coordinate the fix and release process.
+                The problem is confirmed and a list of all affected versions is determined.
+                Code is audited to find any potential similar problems.
+                Fixes are prepared for all releases which are still under maintenance.
+                These fixes are not committed to the public repository but rather held locally or in a private fork pending the announcement.</li>
+
+              <li>A suggested embargo date for this vulnerability is chosen and a CVE
+                (Common Vulnerabilities and Exposures (CVE®)) is requested for the vulnerability,
+                through the GitHub Advisory database.</li>
+
+              <li>A prenotification may be published on the security announcements channels listed below,
+                providing information about affected projects, severity, and the embargo date.</li>
+
+              <li>On the embargo date, the announcement is published.
+                The changes are pushed to the public repository and new builds are deployed.
+                <!--Within 6 hours of the mailing list being notified, a copy of the advisory will be published on the Bytecode Alliance blog.-->
```

Comment:
```suggestion

```
Looks like a leftover fragment of the original security policy. Do we need to keep it?